docs/install_instructions.md

```diff
@@ -0,0 +1,95 @@
+## Installation
```

Comment:
can you rename this file to "INSTALL.md"? so it's more standardized with other projects